app/controllers/admin/orders_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Admin::OrdersController < Admin::AdminsController
+  def index
+    @orders = Order.includes(:order_products, :products)
```

Comment:
```suggestion
    @orders = Order.includes(order_products: :product)
```
ri chứ nhỉ từ order em gọi order detail rồi mới gọi product mà

Reply:
Dạ em chưa hiểu sâu cái chỗ ni lắm, lên đọc document rồi làm theo thôi. Để em xem lại thử

Reply:
Dạ được rồi chị. :D Em cám ơn